circle.yml

```diff
@@ -6,13 +6,7 @@ test:
     - NODE_ENV=test make client/config/index.js:
         parallel: true
   override:
-    - bin/run-lint :
-        parallel: true
```

Comment:
Yes, it’s important to run on `master`, too – you never know what will happen after the merge. Also, need to make sure we run in on direct commits to `master` (I think we have that disabled, but still, better safe than sorry).